### PR TITLE
Add OpsLevel extension

### PR DESCRIPTION
**Pull request by DuckyC**
## Description

This new extension adds the possibility to search through your OpsLevel services.
It allows you to open attached resources like repositories, slack, documentation. 

## Screencast

https://user-images.githubusercontent.com/3424826/229613778-3adea28c-bd59-48cc-b5b7-08443613a83e.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/opslevel/src/clone-action.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                    ) : null,
+                    <ActionPanel.Submenu
+                        key="open-in-choice"
+                        title="Open in ..."
+                        isLoading={isApplicationsLoading}
+                        onOpen={onVisisted}
+                    >
+                        {applications?.map((app) => (
+                            <Action
+                                key={`open-in-${app.bundleId}`}
```
Comment (sxn) [inline extensions/opslevel/src/clone-action.tsx:108]: You can use [FileIcon](https://developers.raycast.com/api-reference/user-interface/icons-and-images#fileicon) (`icon={{ fileIcon: app.path }}`) to show the app's icon :D

**Review by sxn — CHANGES_REQUESTED** [review]
thanks for the PR! Overall it looks good, though I couldn't test it because I don't have an OpsLevel account, and it's…taking a while to create a trial account 😅

One thing i noticed is that some actions have icons and some don't, which doesn't conform to the [guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store#action-panel). Could you ensure either all actions have an icon or none of them do?

File: extensions/opslevel/src/clone-action.tsx (hunk 1 above)
Comment (DuckyC) [reply extensions/opslevel/src/clone-action.tsx:111]: Great trick!

File: extensions/opslevel/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    const { isLoading, error, data, revalidate } = useCachedPromise(fetchServices, [], {
+        keepPreviousData: true,
+    });
+
+    const { services: visitedServices, visitService, isLoading: isLoadingVisistedServices } = useVisitedServices();
+
+    if (lastRefresh < Date.now() - 1000 * 60 * 5) {
+        revalidate();
+        lastRefresh = Date.now();
+    }
```
[the remark this answers is not in the document]
Comment (DuckyC) [reply extensions/opslevel/src/index.tsx]: Hmm, i was a bit confused about the documentation. But i'm very happy to hear thats how it works!

**Pull request merged**